### PR TITLE
Use Sample service bulk ACL update method 

### PR DESCRIPTION
**Pull request by MrCreosote**
Should hopefully speed up setting ACLs for an object with a lot of
sample IDs.

Note that the sample service integration tests required no changes.

### CODE REVIEW

File: python_dependencies/Pipfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ requests = "==2.22.0"
 # handle service dependencies
 cachetools = "*"
 configparser = "*"
-pymongo = "*"
+pymongo = "==3.11.0"
```
Comment (MrCreosote) [inline python_dependencies/Pipfile:16]: Originally pipenv auto upgraded to v4.X which cause the handle service tests to start failing, so I pinned it and the test magically started working again. 
@Tianhao-Gu heads up

File: python_dependencies/Pipfile [hunk 2]
```diff
@@ -30,6 +30,8 @@ jinja2 = "==2.11.2"
 gevent = "==20.5.0"
 gunicorn = "==20.0.4"
 kafka-python = "==2.0.1"
+greenlet = "==0.4.16"
```
Comment (MrCreosote) [inline python_dependencies/Pipfile:33]: This one's a bit weird - if you examine the lock file you'll see that `greenlet` was already installed and all the hashes are the same. However, the new version of the sample service was stuck constantly restarting the workers with a warning about `greenlet` binary compatibility and explicitly installing `greenlet` made it start so *shrug*

**Review by dakotablair — APPROVED** [review]
Looks good to me, seems simpler over all.

**Review by bio-boris — APPROVED** [review]
LGTM

**Pull request merged**